locales/en/plugin__odf-console.json

```
@@ -593,6 +593,7 @@
   "ObjectBucket details": "ObjectBucket details",
   "ObjectBucketClaims": "ObjectBucketClaims",
   "Attach to Deployment": "Attach to Deployment",
+  "Disabled because the Object Bucket Claim is being deleted.": "Disabled because the Object Bucket Claim is being deleted.",
```

```suggestion
  "Disabled because the ObjectBucketClaim is being deleted.": "Disabled because the ObjectBucketClaim is being deleted.",
```

Ref: http://openshift.github.io/openshift-origin-design/conventions/documentation/capitalization.html

---

ACK